### ISSUE TITLE
_ is not defined

### ISSUE THREAD

**Issue by abunour22** [angular-ui/angular-google-maps#1966]
I have these files But i got this error 
        lodash.min.js
        angular-google-maps.min.js

Uncaught ReferenceError: _ is not defined

So what i missed ?!

**Comment by kamil-mrzyglod** [comment]
@abunour22 Are you sure lodash is loaded before Angular Google Maps?

**Comment by mfp22** [comment]
I started on a project recently with lots of npm package dependencies bundled together, and I think I had this error. It was causing all sorts of weird issues. The fix ended up being installing an earlier version of angular-google-maps. angular-google-maps@2.3.3 worked.

**Comment by abunour22** [comment]
Yes its lo added before Angular Google Maps but issue still exists !!

And yes i used earlier version of angular-google-maps. but issue still exists !!

**Comment by 108abhishek** [comment]
I had same issue. It just deleted _ from (window, angular, _) . It left it like (window, angular, );
It is working fine now.
